cli-feed-v4.json

```
@@ -24443,6 +24453,379 @@
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x64.4.0.5780.zip",
+          "sha2": "d139149f897a0a39a1c960e71e4bdd2a475e4fda4e1cba14c684c2b421aa1935",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.win-x86.4.0.5780.zip",
+          "sha2": "dc9917152c6becf8919e864933a9d137b52cdf0cc51ac5c1d6776e5fa0aae333",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-x86.4.0.5780.zip",
+          "sha2": "ed3234722696a8f621e3179a644955d2399591d61e25307486ab6ecc8165f7aa",
+          "size": "minified",
+          "default": "true"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "arm64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.5780/Azure.Functions.Cli.min.win-arm64.4.0.5780.zip",
+          "sha2": "a31bd555992af47cebefa6e0e82afbe3831ab1df5cf8b525774fd9e17d4004aa",
+          "size": "minified",
+          "default": "false"
+        }
+      ]
+    },
+    "4.77.0-inprocess": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimes": {
+        "dotnet": {
+          "net8": {
+            "displayInfo": {
+              "displayName": ".NET 8.0",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET",
+              "description": "LTS",
+              "endOfLifeDate": "2026-11-12T00:00:00Z"
+            },
+            "capabilities": "net8",
+            "sdk": {
+              "name": "Microsoft.NET.Sdk.Functions",
+              "version": "4.4.0"
+            },
+            "default": false,
+            "toolingSuffix": "net8-in-process",
+            "localEntryPoint": "func.exe",
+            "targetFramework": "net8.0",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/4.0.3038",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/4.0.3038",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.3.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/dotnet:4-dotnet8",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~4",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+            },
```

Missing the enablement setting.
```suggestion
            "serviceAppSettings": {
              "FUNCTIONS_EXTENSION_VERSION": "~4",
              "FUNCTIONS_WORKER_RUNTIME": "dotnet",
              "FUNCTIONS_INPROC_NET8_ENABLED": "1"
            },
```